starfish/test/full_pipelines/api/test_iss_api.py

```diff
@@ -139,4 +139,4 @@ def test_iss_pipeline_cropped_data(tmpdir):
     assert pipeline_log[3]['method'] == 'PerRoundMaxChannel'
 
     # 28 of the spots are assigned to cell 0 (although most spots do not decode!)
-    assert np.sum(assigned['cell_id'] == '0') == 28
+    assert np.sum(assigned['cell_id'] == '1') == 28
```

Comment:
Why did this change to cell_id 1?

Reply:
We are finding more cells now than before because we had to turn off border exclusion.  If we turn on border exclusion with 3D and it has a single zplane, it effectively zeroes out the entire image!

Once https://github.com/scikit-image/scikit-image/pull/4325 is released, we can go back to the previous behavior.